picard/config_upgrade.py

```diff
@@ -427,6 +427,17 @@ def rename_option(config, old_opt, new_opt, option_type, default):
         _s[new_opt] = _s.value(old_opt, option_type, default)
         _s.remove(old_opt)
 
+        _p = config.profiles
+        ListOption.add_if_missing("profiles", "user_profiles", [])
+        Option.add_if_missing("profiles", "user_profile_settings", {})
```

Comment:
Since those are also used in `SettingConfigSection.__init__()`, we should move them to a class method and call this method.

Something like:

```python
class SettingConfigSection(ConfigSection):
    ....
    @classmethod
    def init_profile_options(cls):
        ListOption.add_if_missing("profiles", cls.PROFILES_KEY, [])
        Option.add_if_missing("profiles", cls.SETTINGS_KEY, {})
```

Untested though.